compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -248,6 +259,46 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
     (commonMeta ++ applicationMeta, commonDetails ++ taskSpecificDetails)
   }
 
+  private def listMd(s: String): String = {
```

Comment:
These are redundant with WorkflowCompiler - you can make the protected methods in the shared parent class `ExectuableCompiler`.

Reply:
Done.